CHANGES.md

```diff
@@ -1,6 +1,362 @@
 Release Changes List
 ====================
 
+
+version 1.29.0
+==============
+
+released December 15, 2022
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* compile times have been reduced for most platforms via the use of 'jemalloc'
+* reduced overheads and improved scalability of creating distributed arrays
+* made significant additions and improvements to the 'IO' module
+* added a new compilation error framework and improved several error messages
+* improved choices made by procedure overload resolution to improve usability
+* added an experimental 'weak pointer' type with the goal of getting user input
+* added a prototype mode to capture the generated assembly for a given routine
+* made numerous renamings, deprecations, and improvements to standard modules
+* fixed a number of user-reported bugs
+
+Packaging / Configuration Changes
+---------------------------------
+* made our 'Dockerfile' use the current source rather than a specific release
+* LLVM 14 is now required when using the LLVM back-end on a Mac OS X  
+  (see https://chapel-lang.org/docs/1.29/usingchapel/prereqs.html)
+* doxygen 1.8.17 is now required to make the compiler developer documentation  
+  (see https://chapel-lang.org/docs/1.29/usingchapel/prereqs.html)
```

Comment:
Only noting because it's so close to top, this seems minor to me and I might move to "Developer-oriented changes: 'dyno' Compiler improvements/changes". It also reads like 1.8.17 exactly is required, but I'm guessing at least the version is required?

Reply:
Michael and I discussed that and went back and forth and then I got lazy, but since you said it, I'll definitely move it (he didn't disagree, as I understood it).